<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1026</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="add_bread">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>10</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить хлеб</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ask_to_delete">
    <property name="geometry">
     <rect>
      <x>788</x>
      <y>340</y>
      <width>201</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Попросить сотрудницу удалить </string>
    </property>
   </widget>
   <widget class="QPushButton" name="print_a_check">
    <property name="geometry">
     <rect>
      <x>868</x>
      <y>380</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Распечатать чек</string>
    </property>
   </widget>
   <widget class="QPushButton" name="end_the_day">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>460</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Закончить день</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buy_coffee">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>420</y>
      <width>161</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Купить кофе</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_oil">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>50</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить масло</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_water">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>90</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить воду</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_meat">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>130</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить мясо</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_pasta">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>170</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить макароны</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>100</y>
      <width>501</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_4">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>490</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="start_the_day">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>500</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Начать день</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1026</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
